keyboards/crkbd/rules.mk

```diff
@@ -2,7 +2,4 @@
 #   change yes to no to disable
 #
 
-DEFAULT_FOLDER = crkbd/rev1
-
-RGBLIGHT_SUPPORTED = yes
-RGB_MATRIX_SUPPORTED = yes
+DEFAULT_FOLDER = crkbd/rev2
```

Comment:
Changing the default folder would have to go via `develop`.

Reply:
revert https://github.com/qmk/qmk_firmware/pull/22676/commits/e08935278640555270da85fef45fc68328d0a9c1.
I'll change the default folder after developing rev2 and create a PR to `develop` fist.